src/python_routing_v01/compute_nhd_routing_SingleSeg.py

```diff
@@ -1029,23 +1029,31 @@ def main():
     # initialize flowveldepth dict
     qlateral = {connection: {"qlatval": [],} for connection in connections}
 
-    if ( 1==1
+    if (
+        1
+        == 1
         # run_route_and_replace_test
     ):  # test 2. Take lateral flow from wrf-hydro r&r output
-        ql_input_folder = r"/home/APD/inland_hydraulics/wrf-hydro-run/OUTPUTS"
-        ql_files = glob.glob(ql_input_folder + "/*.CHRTOUT_DOMAIN1")
+        # data[supernetwork_data["bottomwidth_col"]]
+        ql_input_folder = supernetwork_data["ql_input_folder"]
```

Comment:
`ql_input_folder = supernetwork_data.get("ql_input_folder",None)` will yield None if the key wasn't created. Then we can follow with logic to replace the value with a command-line argument if one was supplied, or with a default value or an error message.

Reply:
This applies to all of the paths being passed to the function in this section.